config/nvim/keys.vim

```diff
@@ -0,0 +1,7 @@
+" Remap Leader Key
+" ---------------------------------------------------------------------------
+let mapleader = "\<Space>"
+
+" Deoplete Tab Completion
+" ---------------------------------------------------------------------------
+inoremap <expr><TAB>  pumvisible() ? "\<C-n>" : "\<TAB>"
```

Comment:
Possibly.  I use <kbd>TAB</kbd> to expand snippets.  To override it in your config, you have to explicitly unmap it first or my customization should/could be moved into separate files to make it easy to override them.

I'll have to think about it.